### DISCUSSION TITLE
Welcome to gerev Discussions!

### DISCUSSION THREAD

**Discussion by yuvalsteuer** [GerevAI/gerev#87] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ndamix-Store** [comment]
Great

**Comment by Danmasani12** [comment]
hi dear,

please i want to run your app but i discover its saying server is not responding and also the FastAPI is not responding

**Comment by yuvalsteuer** [comment]
Could you provide logs?
